plugins/catalog-backend/migrations/20230525141717_stitch_queue.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// @ts-check
+
+/**
+ * @param { import("knex").Knex } knex
+ */
+exports.up = async function up(knex) {
+  await knex.schema.alterTable('refresh_state', table => {
```

Comment:
I'm a little worried about how long an alterTable like this will take on a big database - it may cause deployments to time out?

I _think_ it's safe because the column is nullable and doesn't have a default value,  so it should be fast.

Reply:
Yep, in my tests it seems to be fast - indeed I did intentionally make sure to keep the nullability both in the table and the index.